src/shared_utils.hpp

```diff
@@ -20,8 +20,6 @@ floatType _somaSurface(const SomaType type,
                        const ContainerDiameters& diameters,
                        const ContainerPoints& points) {
     size_t size = points.size();
-    if (size == 0)
```

Comment:
This one to discuss. Because of it, SOMA_UNDEFINED becomes unreachable further in code => morphologies without soma have soma surface 0. However how correct to have 0 surface if there is no soma at all?

Reply:
I think raising is good. This is clearer and more correct imao.